extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcJsonWebTokenProducer.java

```diff
@@ -79,7 +79,7 @@ private JsonWebToken getTokenCredential(Class<? extends TokenCredential> type) {
             return new OidcJwtCallerPrincipal(jwtClaims, credential);
         }
         String tokenType = type == AccessTokenCredential.class ? "access" : "ID";
-        LOG.tracef("Current identity is not associated with an %s token", tokenType);
+        LOG.warnf("Current identity is not associated with an %s token", tokenType);
```

Comment:
I wonder if we shouldn't add more context to the message (like, saying what should be done to fix it). As a user, I wouldn't know what to do when this appears in the logs

Reply:
Good point,  will do a bit later,  thanks

Reply:
I've tried to make it more informative, it can be much more verbose, but hopefully the current message is providing just enough info to explore further